astroquery/oac/core.py

```diff
@@ -476,25 +477,15 @@ def _parse_result(self, response, verbose=False):
         if not verbose:
             commons.suppress_vo_warnings()
 
-        try:
-            if response.status_code != 200:
-                raise AttributeError
+        if response.status_code != 200:
```

Comment:
We could simply use `response.raise_for_status()` as we do in the other modules.